### ISSUE TITLE
New Logo Design For Baking App

### ISSUE THREAD

**Issue by zularizal** [kalxasath/Baking-App#1]
Hi, I am a graphic designer, I want to help you in graphic design.

After I review your project, you have no logo on this project yet. Therefore I want to contribute to this project by creating a new logo / icons. What do you think?

**Comment by kalxasath** [comment]
Hello Zularizal,

that's fantastic, thank you for your contribution!

**Comment by kalxasath** [comment]
Thaks a lot!

Very interesting the logos, if it is not hard to you, I would like the middle by replacing the bread with the cooking hat. Also, can you provide this logo for the dimensions that an Android application wants?

**Comment by zularizal** [comment]
Sure, do you want to make it adaptive or just an icon?

**Comment by zularizal** [comment]
How about this?

![g9730](https://user-images.githubusercontent.com/38547278/40776820-1425a3b0-64f6-11e8-9032-ebf6bcee3b99.png)

**Comment by kalxasath** [comment]
Very nice, I like it!

Thank you!

**Comment by zularizal** [comment]
Wait a minute, I'll upload it to drive

**Comment by kalxasath** [comment]
Please can you make the icons for the launcher according to Android specifications, because I can't use any graphics program to adjust the size

**Comment by zularizal** [comment]
Sure, i will replace new icons to repository..

**Comment by zularizal** [comment]
Hi @kalxasath , I've uploaded logo and icon files to drive
https://drive.google.com/open?id=15Bovm9yhc-GYk7W-hp1biBmxw5dup94_

and also I have sent a pull request for you, please check #2

**Comment by kalxasath** [comment]
Hello @zularizal,

I've merged your pull request, you work is fantastic, thank you very much for those beautiful icons & logo!

**Comment by kalxasath** [comment]
Hello Zularizal,

I have a new project under the github https://github.com/kalxasath/Capstone
This is a vehicle expense tracking app, I would appreciate if you have time and are interested to make app icons for this app.

Please let me know if you have interest in this project.

Thank you

**Comment by zularizal** [comment]
Of course, tomorrow I will make an icon for this project or the day after tomorrow.

**Comment by kalxasath** [comment]
Thank you very much ;)
